packages/helpermodules/command.py

```diff
@@ -335,7 +335,7 @@ def addComponent(self, connection_id: str, payload: dict) -> None:
             "."+payload["data"]["deviceType"]+"."+payload["data"]["type"], "modules")
         component_default = component.get_default_config()
         component_default["id"] = new_id
-        if payload["data"]["type"] == "counter":
+        if "counter" in payload["data"]["type"]:
```

Comment:
In Z. 571 wird auch auf `type == 'counter'`geprüft. Und in Z. 576 wird der Typ in das Topic eingesetzt, ähnlich wie Du es in fault_state.py schon korrigiert hast.

Reply:
Ich schaue es mir an.